__tests-pacts__/serlo.org-database-layer/subscriptions.ts

```diff
@@ -32,7 +32,10 @@ test('Subscriptions', async () => {
     path: `/subscriptions/${user.id}`,
     body: {
       userId: user.id,
-      subscriptions: Matchers.eachLike({ id: Matchers.integer(1) }),
+      subscriptions: Matchers.eachLike({
+        id: Matchers.integer(1),
+        sendEmail: Matchers.boolean(true),
```

Comment:
This should probably be
```suggestion
        sendEmail: Matchers.boolean(false),
```
Since you post `sendEmail: false`. Doesn't matter much, though, since you don't do anything with the response anyways.